src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -957,10 +953,14 @@ private String getFindByIndexBody(String selectorJson,
         return finalbody.toString();
     }
 
-    HttpResponse executeRequest(HttpRequestBase request) {
+    InputStream executeRequest(HttpConnection request) {
```

Comment:
`Search` class uses this in the `queryForStream` method and does not have a `client` object.